docs/install/install-methods.md

```diff
@@ -1,17 +1,90 @@
 ---
 id: install-methods
-title: Installation Methods
+title: Installation
 sidebar_position: 2
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Installation Methods
 
-Vignette provides a lot of official ways for you to get the software. We have listed what we support so far below, however, if you don't see your
-target platform, please [raise an issue](https://github.com/vignetteapp/vignette/issues/new) for your platform and see if we can support it.
+Vignette provides a lot of official ways for you to get the software. We have listed what we support so far below, however, if you don't see your target platform, please [raise an issue](https://github.com/vignetteapp/vignette/issues/new) for your platform and see if we can support it.
+
+<Tabs groupId="operating-systems">
+  <TabItem value="microsoft-store" label="Microsoft Store (Windows)" default>
+  
+  Vignette is Coming Soon to the Microsoft Store! 
+  
+  
+  Until then, please consinder installing it via our Portable Installation Guide
+  </TabItem>
+  <TabItem value="steam" label="Steam">
+  Vignette is Coming Soon to Steam!
+  
+
+  Until then, please consinder installing it via our Portable Installation Guide or AUR Installation Guide.
+  </TabItem>
+  <TabItem value="aur" label="AUR (Linux)">
+
+  If you prefer, to install Vignette on Linux we provide the official [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/) AUR package. This guide will be using `yay`.
+
+
+
+## Installation process
+
+In order to install Vignette using yay, run the following command:
+
+`yay vignette-git`
+
+
+
+If more than one package appears you should find the right package, which should be listed as `aur/vignette-git`. In this example, that package is number `1`.
+
+![Image of yay listing package, with aur/vignette-git on position 1](/img/install/yay-terminal-window.webp)
+
+
+
+:::note 
+
+When prompted for your password for sudo, enter it to proceed with the installation.
+
+:::
```

Comment:
I'm not sure if this is necessary since it should be already clear (and you're speaking to Linux users lol)

Reply:
Was already there... I kept the page as it was.

Reply:
I think it could be removed but it's also fine to keep it (maybe change the wording to something like suggestion)

```suggestion
:::note 

When prompted, enter your password so it could proceed with the installation.

:::
```

Reply:
I think that's out of scope for this PR, but we can do that on Ayyrik's branch directly

Reply:
Sounds fine to me